src/IntegrationTests/java/uk/gov/hmcts/reform/sscs/sya/SyaEndpointsIt.java

```diff
@@ -98,7 +118,7 @@ public class SyaEndpointsIt extends AbstractEventIt {
     private MockMvc mockMvc;
 
     @Autowired
-    private SubmitAppealService submitAppealService;
+    private SubmitAppealServiceInterface submitAppealService;
```

Comment:
Remove the interface bit in name, just a naming convention thing. May be you don't need Interface as well.

Reply:
Removed the interface itself and replace with SubmitAppealServiceBase